backend/blueprint/api/utils/response.py

```diff
@@ -13,3 +13,10 @@ def generateMessage(self, message):
         self.response['message'] = self.message
 
         return self.response
+
+    def generateURL(self, message, URL):
+        self.URL = URL
```

Comment:
Need for extra self parameter. We can directly add it to the response and needed can parsed from the response object.

Reply:
`self.response['url'] = URL`

Also why the over head of storing the values in the different variable, when it can be parsed from the response entity.